src/components/display/components/DisplayListComponent.tsx

```diff
@@ -0,0 +1,54 @@
+import { StyleSheet, View } from 'react-native'
+import React, { useMemo, useRef } from 'react'
+import { DisplayListArgs, DisplayListItem } from 'display-api'
+import { FlatList } from 'react-native-gesture-handler'
+import { ComponentDelegate } from '../ComponentDelegate'
+import { IconButton } from 'react-native-paper'
+import colors from '../../../utils/colors'
+import { BottomMenu } from '../../blockMenu/bottomMenu'
+
+export const DisplayListComponent = ({ items }: DisplayListArgs) => {
+	const menuRefs = useMemo(() => Array.from({ length: items.length }, () => useRef(null)), [items])
+
+	const renderItem = ({ item, index }: { item: DisplayListItem; index: number }) => {
+		return (
+			<>
+				<View style={styles.item}>
+					<IconButton
+						disabled={!item.menu}
+						style={styles.rightIcon}
+						color={colors.primary}
+						icon='dots-horizontal'
+						onPress={() => {
+							menuRefs[index].current?.handleOpen()
+						}}
+					/>
+					<ComponentDelegate component={item.component} />
+				</View>
+				{item.menu && <BottomMenu ref={menuRefs[index]} menu={item.menu} />}
```

Comment:
It looks like rendering icons in custom menu items is broken. They all seem to display `Box`, when I specified `Plus`, etc.